### DISCUSSION TITLE
Web interface login (too obvious a question, but...)

### DISCUSSION THREAD

**Discussion by kubycsolutions** [automatic-ripping-machine/automatic-ripping-machine#1144] [Q&A] [answered]
Where are the web UI's user name and password set? They don't seem to be those of the arm account, and I don't see them as parameters in the setup script.

**Comment by microtechno9000** [comment]
the user is always admin, the password is set on the GUI, on the password reset page

**Reply by kubycsolutions** [reply, marked as answer]
Ah. admin'password. Sorry, must have been asleep at the keyboard.

**Comment by ghost** [comment]
is it possible to change the username? For my server i have several internal apps that use the same username for ease and it would be nice to make ARM match them.
